inc/template-functions.php

```diff
@@ -106,10 +106,7 @@ function twentynineteen_can_show_post_thumbnail() {
  * Returns true if image filters are enabled on the theme options.
  */
 function twentynineteen_image_filters_enabled() {
-	if ( 'inactive' === get_theme_mod( 'image_filter' ) ) {
-		return false;
-	}
-	return true;
+	return 'inactive' === get_theme_mod( 'image_filter' );
```

Comment:
This seems flipped to me. It'll now return true when it should be false.

Reply:
Yes, you are right. It came to me after I had created the PR and closed the computer.